Content.Client/Administration/UI/Tabs/ObjectsTab/ObjectsTab.xaml.cs

```diff
@@ -27,6 +29,9 @@ public sealed partial class ObjectsTab : Control
     private readonly TimeSpan _updateFrequency = TimeSpan.FromSeconds(2);
     private TimeSpan _nextUpdate;
 
+    private Action<NetEntity>? TeleportToObjAction { get; set; }
+    private Action<NetEntity>? DeleteObjAction { get; set; }
+
```

Comment:
Subscribing to an object's own event makes no sense, just subscribe to it on the child.